data-prepper-plugins/geoip-prepper/src/test/java/com/amazon/dataprepper/plugins/prepper/geoip/provider/GeoIpProviderFactoryTest.java

```diff
@@ -0,0 +1,62 @@
+package com.amazon.dataprepper.plugins.prepper.geoip.provider;
+import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.plugins.prepper.geoip.GeoIPPrepperConfig;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.internal.matchers.apachecommons.ReflectionEquals;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
I see `@RunWith(MockitoJUnitRunner.class)` in another test class, can we keep them consistent and just one type of annotation?